src/Backend/Modules/Faq/Ajax/SequenceQuestions.php

```diff
@@ -35,16 +35,14 @@ public function execute(): void
 
         // is the question moved to a new category?
         if ($fromCategoryId != $toCategoryId) {
-            $item = [];
-            $item['id'] = $questionId;
+            $item = BackendFaqModel::get($questionId);
```

Comment:
Don't change anything in the actions, only in the model.

Reply:
We need the full question to be able to update it

Reply:
Fixed 232348709